File: nationsFrontEnd/src/app/shared/models/Language.ts
export class Language{

  constructor(private _languageId?: number,
              private _language?: string) {
  }

  get languageId(): number {
    return this._languageId;
  }

  set languageId(value: number) {
    this._languageId = value;
  }

  get language(): string {
    return this._language;
  }

  set language(value: string) {
    this._language = value;
  }
}

File: nationsBackEnd/src/main/java/com/qualco/nations/models/Language.java
package com.qualco.nations.models;

import lombok.Data;

import javax.persistence.*;
import java.io.Serializable;

@Data
@Entity
@Table(name = "languages")
public class Language implements Serializable {

    private static final long serialVersionUID = 1L;

    @Id
    @Column(name = "language_id", nullable = false)
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer languageId;

    @Column(name = "language", nullable = false)
    private String language;

}
